targets/python/templates/PlayFabUtil.py.ejs

```diff
@@ -0,0 +1,59 @@
+# using PlayFab.Json;
+import json
+<% if (hasClientOptions) {
+%>import PlayFabClientModels
+<% } else if (hasServerOptions) {
+%>import PlayFabServerModels
+<% } %>
+
+class PlayFab:
+    [AttributeUsage(AttributeTargets.Field, AllowMultiple = false)]
```

Comment:
This is C#
I'll have to show you how to use meta-tags in python.
I'll have to remember how to use meta-tags in python...